source/includes/package-repository/_40-update.md.erb

```diff
@@ -0,0 +1,116 @@
+## Update a package repository
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/repositories/0530f1a9-5282-4b69-be59-066a35076452' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json' \
+      -H 'If-Match: "26b227605daf6f2d7768c8edaf61b861"' \
+      -H 'Content-Type: application/json' \
+      -X PUT -d '{
+        "name" : "npmjs.org",
```

Comment:
Put in an id. The implementation was changed to not allowing id to be renamed. Otherwise for update, there would have been a new id each time.